pkg/util/conditions/condition_test.go

```diff
@@ -35,7 +36,7 @@ func TestSetCondition(t *testing.T) {
 		input   *operatorv1.OperatorCondition
 
 		expected arov1alpha1.ClusterStatus
-		wantErr  error
+		wantErr  string
```

Comment:
I was wondering how the type could change and none of its usage, we can delete `wantErr` from the struct since it is not used in a useful way.

Reply:
Right. Done. Tks!